src/redux/books/books.js

```diff
@@ -0,0 +1,18 @@
+import Buttons from '../../components/Buttons';
+
+const Books = () => (
+  <div className="booksContainer">
+    <ul>
+      <li style={{ display: 'flex' }}>
+        <p style={{ margin: '0' }}>No books to display!</p>
+        <Buttons label="Remove" />
+      </li>
+    </ul>
+    <form>
+      <input type="text" placeholder="Enter book name" />
+      <Buttons label="Add" />
+    </form>
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] I think here you can follow the `single responsibility principle` by putting the book info and form into separate components and rendering them in the `books.js` component.

- [ ] Also, please do not forget to use `props` in your new book component to render `No books to display!` from the `books.js` component. In order to make it more clear I am providing with the below screenshot of the project requirements 👇 
![Screenshot (135)](https://user-images.githubusercontent.com/67087664/149827308-d69930bc-bf2b-4abf-b9f0-601987f46d0c.png)

Reply:
Hello, thank you for the review. I can't seem to find the source document of the screenshot you have sent though... I seem to be having a completely different file for this project. 
here is the link to the one I am currently doing https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md
![project](https://user-images.githubusercontent.com/47208108/149830384-01e9f415-0502-4e7a-aa3f-290233431fbd.png)

Reply:
Good morning @elyor-doniyorov , I am going to resubmit my work for approval because of time. I have refactored the book component into two as suggested. As for the project requirements, I have seen that some of them such as creating the author and title properties are in my next milestone. Thanks again for your review

Reply:
Good morning @Mosams, thanks for explaining.